src/WebJobs.Extensions.Http/HttpTriggerAttributeBindingProvider.cs

```diff
@@ -280,7 +280,7 @@ internal static void ApplyBindingData(object target, IDictionary<string, object>
             {
                 // apply binding data from request body if present
                 var bindingData = new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase);
-                if (request.Body.Length > 0)
```

Comment:
This change was driven by an issue caused by a mocked request that wasn't setup as expected. Reverting.

Reply:
Thanks for fixing